aries-site/src/data/structure.js

```diff
@@ -398,6 +398,7 @@ export const structure = [
     seoDescription:
       'When the user needs to select one or more options, use a checkbox.',
     sections: [],
+    relatedContent: ['RadioButtonGroup'],
```

Comment:
```suggestion
    relatedContent: ['Forms', 'RadioButtonGroup', 'Select'],
```
These might me good related items -- all about selecting from a set of options

Reply:
good thinking when I saw forms on your `textinput` i thought on here as well